bip-musig2.mediawiki

```diff
@@ -416,17 +416,17 @@ Input:
 * Run ''PartialSigVerifyInternal(psig, pubnonce<sub>i</sub>, pk<sub>i</sub>, session_ctx)''
 * Return success iff no failure occurred before reaching this point.
 
-'''''PartialSigVerifyInternal(psig, pubnonce, pk<sup>*</sup>, session_ctx)''''':
+'''''PartialSigVerifyInternal(psig, pubnonce, pk, session_ctx)''''':
 * Let ''(Q, gacc, _, b, R, e) = GetSessionValues(session_ctx)''; fail if that fails
 * Let ''s = int(psig)''; fail if ''s &ge; n''
-* Let ''R<sup>*</sup><sub>1</sub> = cpoint(pubnonce[0:33]), R<sup>*</sup><sub>2</sub> = cpoint(pubnonce[33:66])''
-* Let ''R<sup>*</sup>' = R<sup>*</sup><sub>1</sub> + b⋅R<sup>*</sup><sub>2</sub>''
-* Let ''R<sup>*</sup> = R<sup>*</sup>' '' if ''has_even_y(R)'', otherwise let ''R<sup>*</sup> = -R<sup>*</sup>' ''
-* Let ''P = cpoint(pk<sup>*</sup>)''; fail if that fails
+* Let ''R<sub>⁎,1</sub> = cpoint(pubnonce[0:33]), R<sub>⁎,2</sub> = cpoint(pubnonce[33:66])''
+* Let ''Re<sub>⁎</sub>' = R<sub>⁎,1</sub> + b⋅R<sub>⁎,2</sub>''
+* Let ''Re<sub>⁎</sub> = Re<sub>⁎</sub>' '' if ''has_even_y(R)'', otherwise let ''Re<sub>⁎</sub> = -Re<sub>⁎</sub>' ''
```

Comment:
may have a better rendering, I haven't checked.
```suggestion
* Let ''Re'<sub>⁎</sub> = R<sub>⁎,1</sub> + b⋅R<sub>⁎,2</sub>''
* Let ''Re<sub>⁎</sub> = Re'<sub>⁎</sub> '' if ''has_even_y(R)'', otherwise let ''Re<sub>⁎</sub> = -Re'<sub>⁎</sub> ''
```

on the other hand, it's probably more "systematic" to put the prime afterwards, even though the rendering is strange.

in any case, we should probably make the position of the prime consistent here and in the `k`s in Sign, which also have a subscript and a prime.

Reply:
Made position of prime in k consistent